index.js

```diff
@@ -1,17 +1,37 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    (this.items = []), (this.length = this.items.length);
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.items.length) throw new Error("OutOfBounds");
+    return this.items.indexOf(pos);
+  }
 
-  max() {}
+  max() {
+    if (!this.items.length) throw new Error("EmptySortedList");
+    return Math.max(...this.items);
+  }
 
-  min() {}
+  min() {
+    if (!this.items.length) throw new Error("EmptySortedList");
+    return Math.min(...this.items);
+  }
 
-  sum() {}
+  sum() {
+    return this.items.reduce((acc, value) => (acc += value), 0);
+  }
 
-  avg() {}
+  avg() {
+    if (!this.items.length) throw new Error("EmptySortedList");
+    return this.items.reduce((acc, value) => (acc += value), 0) / this.length;
```

Comment:
The only thing here would have been to reuse the sum method. no need to reinvent the wheel, or in this case, rewrite the sum method.
```js
return this.sum() / this.length
```